src/main/java/com/es/phoneshop/web/CartPageServlet.java

```diff
@@ -0,0 +1,87 @@
+package com.es.phoneshop.web;
+
+import com.es.phoneshop.model.cart.Cart;
+import com.es.phoneshop.model.cart.exception.OutOfStockException;
+import com.es.phoneshop.model.product.ArrayListProductDao;
+import com.es.phoneshop.model.product.Product;
+import com.es.phoneshop.model.product.ProductDao;
+import com.es.phoneshop.service.CartService;
+import com.es.phoneshop.service.DefaultCartService;
+import com.es.phoneshop.web.exception.ParseProductFromRequestException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.text.NumberFormat;
+import java.text.ParseException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CartPageServlet  extends HttpServlet {
```

Comment:
Here are (maybe) in other places - please reformat the code, remove duplicate spaces, add missing spaces: Ctrl + Alt + L in IntelliJ IDEA.